README.md

```diff
@@ -100,6 +100,49 @@ You can also use `LocalGitHubOAuthenticator` to map GitHub accounts onto local u
 
 You can use your own Github Enterprise instance by setting the `GITHUB_HOST` environment variable.
 
+### GitHub-specific features
```

Comment:
Hi @athornton,
 
Let's make this a separate markdown file `github_scope.md`, and add one sentence here linking to the new document.

Keeping in mind that we have a range of user experience levels, I think it would be best to keep the `README.md` focused on the basics. Additional documents can provide more detailed/advanced use cases. If needed over time, we could create Sphinx documentation.